README.md

```diff
@@ -216,3 +227,115 @@ Gets tax account information for a given nino and tax year
     }
 ```
 
+#### Get All Details
+Gets all information for a given nino and tax year. This includes:
+- all employments
+- for each employment that has any, a list of company benefits
```

Comment:
does not read well, "for each employment that has any" any what might as well add the two words "company benefits" if thats what it is. same for  lines below